### PR TITLE
PR: Added Avatar

### PR DESCRIPTION
**Pull request by Jayesh3636**
- Added Avatar Component : 
  -  Avatar with Picture
  -  Avatar with Text

Kindly review and and let me know the changes 

https://user-images.githubusercontent.com/91378157/153620396-e0bfbfe3-0950-472f-b061-ae79aa502350.mp4

### CODE REVIEW

File: Components/avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+.avatar__text {
+    margin: 3.5rem 0rem;
+}
+
+.avatar__text-examples {
+    margin-top: 2.5rem;
+    display: flex;
+    gap: 3rem;
+    /* justify-content: space-around; */
```
Comment (Koushith) [inline Components/avatar/avatar.css]: you can remove the comments if it's not required.

**Review by Koushith — APPROVED** [review]
LGTM- good to merge.

File: Components/avatar/avatar.css (hunk 1 above)
Comment (Jayesh3636) [reply Components/avatar/avatar.css]: done!

**Review by pavan6115 — APPROVED** [review]
Looks good 👍

**Pull request merged**